pkg/logging/logging.go

```diff
@@ -34,6 +34,7 @@ const (
 	Mutator               = "mutator"
 	DebugLevel            = 2 // r.log.Debug(foo) == r.log.V(logging.DebugLevel).Info(foo)
 	ExecutionStats        = "execution_stats"
+	ViolationMetadata     = "violation_metadata"
```

Comment:
We should be consistent with how audit handles the `details` field:

https://github.com/open-policy-agent/gatekeeper/blob/b7230e0bdc3479b867acd2fd0d7192b1182c59d4/pkg/audit/manager.go#L1115

Reply:
Oh I missed that! 🤦🏼 

Deduping this comment with the `audit` logging. I figured that logging the details of the rego evaluations for every object for every audit run would be too noisy. At present, we only log details for a `violation` in audit so this should be consistent w the audit behavior now:

https://github.com/open-policy-agent/gatekeeper/blob/b7230e0bdc3479b867acd2fd0d7192b1182c59d4/pkg/audit/manager.go#L1106-L1131